docs/technologies.md

```diff
@@ -466,25 +467,29 @@ workflow = Workflow(
         schedule_interval='@once')
 ```
 
-#### Label table
+#### Labels
 
-The `add_label` function allows your workflow to create/overrides a label for a BigQuery table.
+The `table_labels` and `dataset_labels` parameters allows your workflow to create/overrides a label for a BigQuery table and dataset. 
```

Comment:
pass it through Grammarly, I'm not sure if it's grammatically correct

Reply:
Done, but for better correction Grammarly Premium is needed :D